tests/plugins/autocomplete/manual/removematch.md

```diff
@@ -0,0 +1,23 @@
+@bender-tags: 4.10.2, bug, 2194
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, basicstyles, autocomplete, textmatch
+@bender-include: _helpers/utils.js
+
+1. Open the console.
+1. Focus the editor.
+1. Type `@`.
+1. Accept `@anna` dropdown option.
+1. Start deleting inserted match using `backspace` key.
```

Comment:
The important bit of information is missing: steps should be repeated several times (3-5), as error occurs randomly.